src/Annotation/ApiResource.php

```diff
@@ -116,6 +116,16 @@ final class ApiResource
      */
     public $subresourceOperations;
 
+    /**
+     * @var bool
+     */
+    public $isInterface;
+
+    /**
+     * @var array
+     */
+    public $implements;
+
```

Comment:
This should probably be under the `graphql`-key, as this is only supported for the GraphQL format / endpoints.

Reply:
@lukasluecke can you explain `graphql-key` - what does it mean? Prefix for property name?

Reply:
@CvekCoding There is already a configuration key `graphql`, under which we configure the operations etc. - I think it would make sense to include it inside of that, instead of at the root level.